spec/requests/grantees_spec.rb

```diff
@@ -0,0 +1,130 @@
+require "rails_helper"
+
+# This spec was generated by rspec-rails when you ran the scaffold generator.
+# It demonstrates how one might use RSpec to test the controller code that
+# was generated by Rails when you ran the scaffold generator.
+#
+# It assumes that the implementation code is generated by the rails scaffold
+# generator. If you are using any extension libraries to generate different
+# controller code, this generated spec may or may not pass.
+#
+# It only uses APIs available in rails and/or rspec-rails. There are a number
+# of tools you can use to make these specs even more expressive, but we're
+# sticking to rails and rspec-rails APIs to keep things simple and stable.
+
+RSpec.describe "/grantees", type: :request do
+  # Grantee. As you add validations to Grantee, be sure to
+  # adjust the attributes here as well.
+  let(:valid_attributes) {
+    skip("Add a hash of attributes valid for your model")
+  }
+
+  let(:invalid_attributes) {
+    skip("Add a hash of attributes invalid for your model")
+  }
+
+  # describe "GET /index" do
+  #   it "renders a successful response" do
+  #     Grantee.create! valid_attributes
+  #     get grantees_url
+  #     expect(response).to be_successful
+  #   end
+  # end
+
+  describe "GET /show" do
+    let(:grant_id) { FakeIssues.instance.data.first[:id] }
```

Comment:
[suggestion] i think this should be `FakeIssues.instance.data.first[:grantNumber]` to make this a little more clear

Reply:
This is actually wrong, it's not the `id` at all! Thanks for the catch.

Reply:
The FakeIssues is a list of fake data coming from the `/complaints` endpoint, so the top-level ID is the ID of an issue, not a grant or a grantee.

If we want the grant id out of that object, it looks like that should be `first[:attributes][:grantNumber]`. (That being said, I don't think we need to use the FakeIssues to get a grant id, and could probably just use a random identifier, although you'd need to test to make sure)

Reply:
oh ok, is there a `/grantees` endpoint yet? I suppose a `FakeGrantees` source should be created, so we're not going through `FakeIssues`, probably in its own ticket.